packages/fxa-auth-server/lib/payments/paypal-processor.ts

```diff
@@ -310,6 +312,12 @@ export class PaypalProcessor {
     // 4
     const inGracePeriod = this.inGracePeriod(invoice);
     if (!inGracePeriod) {
+      this.log.info('processInvoice', {
+        message: 'Cancelling invoice due to grace period.',
+        invoiceId: invoice.id,
+        created: invoice.created,
+        due: invoice.due_date,
+      });
```

Comment:
Should we add the current date for cancelling invoice?

Reply:
What do you mean?

Reply:
Date to document when the invoice was cancelled.

Reply:
Log messages have timestamps.